instrumented/endpoints.js

```diff
@@ -0,0 +1,95 @@
+  state: 'http://localhost:3001/api/v1/state'
+});
+export default endpoints;
+//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImVuZHBvaW50cy5qcyJdLCJuYW1lcyI6WyJlbmRwb2ludHMiLCJ1c2VycyIsImVsZWN0aW9ucyIsInJlZmVyZW5kdW1zIiwic3RhdGUiXSwibWFwcGluZ3MiOiI7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7QUFlWTs7Ozs7Ozs7O0FBZlosTUFBTUEsU0FBUyw2QkFBRztBQUNoQkMsRUFBQUEsS0FBSyxFQUFFLG9DQURTOztBQUVoQjtBQUNGO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDRUMsRUFBQUEsU0FBUyxFQUFFLHdDQXRDSzs7QUF1Q2hCO0FBQ0Y7QUFDQTtBQUNFQyxFQUFBQSxXQUFXLEVBQUUsMENBMUNHO0FBMkNoQkMsRUFBQUEsS0FBSyxFQUFFO0FBM0NTLENBQUgsQ0FBZjtBQThDQSxlQUFlSixTQUFmIiwic291cmNlc0NvbnRlbnQiOlsiY29uc3QgZW5kcG9pbnRzID0ge1xuICB1c2VyczogJ2h0dHA6Ly9sb2NhbGhvc3Q6MzAwMS9hcGkvdjEvdXNlcnMnLFxuICAvKiBQT1NUIHRvIEFQSTpcbiAgICBSRVFVRVNUIE9CSkVDVDpcbiAgICAgIHtcbiAgICAgICAgZmlyc3RfbmFtZTogJ3N0cmluZycsXG4gICAgICAgIGxhc3RfbmFtZTogJ3N0cmluZycsXG4gICAgICAgIHN0YXRlX25hbWU6ICdzdHJpbmcnLCAqKiB0d28gbGV0dGVyIHN0YXRlIGFiYnYgKipcbiAgICAgICAgZW1haWw6ICdzdHJpbmcnLFxuICAgICAgICBsYW5ndWFnZTogJ3N0cmluZycgKiogJ2VuJyBvciAnZXMnICoqXG4gICAgICB9XG4gICAgUkVTUE9OU0UgT0JKRUNUOlxuICAgICAgU1VDQ0VTU0ZVTExZIENSRUFURUQ6XG4gICAgICB7XG4gICAgICAgIHN1Y2Nlc3M6IFwiWW91IGFyZSBub3cgcmVnaXN0ZXJlZCB0byByZWNlaXZlIG5vdGlmaWNhdGlvbnMgYWJvdXQgdXBjb21pbmcgZWxlY3Rpb25zIGluIHlvdXIgc3RhdGUuIEEgY29uZmlybWF0aW9uIGVtYWlsIGhhcyBiZWVuIHNlbnQgdG8gI3t1c2VyLmVtYWlsfS5cIixcbiAgICAgICAgc3RhdHVzOiAyMDBcbiAgICAgIH1cbiAgICAgIEVNQUlMIERPRVMgTk9UIElOQ0xVREUgJ0AnOlxuICAgICAge1xuICAgICAgICBlcnJvcjogXCJFcnJvciBjcmVhdGluZyBzdWJzY3JpYmVyLiBBIHZhbGlkIGVtYWlsIG11c3QgYmUgcHJvdmlkZWQuXCIsXG4gICAgICAgIHN0YXR1czogNDAwXG4gICAgICB9XG4gICAgICBFTUFJTCBBTFJFQURZIEVYSVNUUzpcbiAgICAgIHtcbiAgICAgICAgZXJyb3I6IFwiI3t1c2VyLmVtYWlsfSBpcyBhbHJlYWR5IHN1c2JzY3JpYmVkIHRvIHJlY2VpdmUgZWxlY3Rpb24gbm90aWZpY2F0aW9ucy5cIixcbiAgICAgICAgc3RhdHVzOiA0MDBcbiAgICAgIH1cbiAgICAgIEdFTkVSQUwgRVJST1I6XG4gICAgICB7XG4gICAgICAgIGVycm9yOiBcIkVycm9yIGNyZWF0aW5nIHN1YnNjcmliZXIuIFBsZWFzZSBlbnN1cmUgYWxsIGVsZW1lbnRzIGluIHRoZSBmb3JtIGFyZSBmaWxsZWQgb3V0IGNvcnJlY3RseS5cIixcbiAgICAgICAgc3RhdHVzOiA0MDBcbiAgICAgIH1cbiAgICAgIFNVQ0NFU1NGVUxMWSBSRU1PVkVEOlxuICAgICAge1xuICAgICAgICBzdWNjZXNzOiBcIllvdSBoYXZlIHN1Y2Nlc3NmdWxseSB1bnN1YnNjcmliZWQgZnJvbSBNaSBWb3RvLCBNaSBWb3oncyBlbWFpbCBsaXN0LiBZb3Ugd2lsbCBubyBsb25nZXIgcmVjZWl2ZSBlbWFpbCBub3RpZmljYXRpb25zIGF0ICN7dXNlci5lbWFpbH0uXCJ9LFxuICAgICAgICBzdGF0dXM6IDIwMFxuICAgICAgfVxuICAqL1xuICBlbGVjdGlvbnM6ICdodHRwOi8vbG9jYWxob3N0OjMwMDEvYXBpL3YxL2VsZWN0aW9ucycsXG4gIC8qXG4gICAgUVVFUlkgUEFSQU1TPyBMT0NBVElPTj8gU1RBVEU/ICBJLkUuIENPP1xuICAqL1xuICByZWZlcmVuZHVtczogJ2h0dHA6Ly9sb2NhbGhvc3Q6MzAwMS9hcGkvdjEvcmVmZXJlbmR1bXMnLFxuICBzdGF0ZTogJ2h0dHA6Ly9sb2NhbGhvc3Q6MzAwMS9hcGkvdjEvc3RhdGUnXG59XG5cbmV4cG9ydCBkZWZhdWx0IGVuZHBvaW50cztcbiJdfQ==
```

Comment:
I didn't see this file being used anywhere in the tests. Can you please explain what this file is for and if it's needed?

Reply:
'Explained' [here](https://github.com/mi-voz-mi-voto/mi-voz-mi-voto/pull/30#discussion_r804866273)